testslide/strict_mock.py

```diff
@@ -153,41 +154,202 @@ class StrictMock(object):
 
     TRIM_PATH_PREFIX = ""
 
-    def __init__(self, template=None, runtime_attrs=None, name=None):
+    _SETTABLE_MAGICS = [
+        "__abs__",
+        "__add__",
+        "__and__",
+        "__bool__",
+        "__bytes__",
+        "__call__",
+        "__ceil__",
+        "__complex__",
+        "__contains__",
+        "__delete__",
+        "__delitem__",
+        "__divmod__",
+        "__enter__",
+        "__enter__",
+        "__eq__",
+        "__exit__",
+        "__exit__",
+        "__float__",
+        "__floor__",
+        "__floordiv__",
+        "__format__",
+        "__ge__",
+        "__get__",
+        "__getformat__",
+        "__getinitargs__",
+        "__getitem__",
+        "__getnewargs__",
+        "__getnewargs_ex__",
+        "__getstate__",
+        "__gt__",
+        "__iadd__",
+        "__iand__",
+        "__ifloordiv__",
+        "__ilshift__",
+        "__imatmul__",
+        "__imod__",
+        "__imul__",
+        "__index__",
+        "__instancecheck__",
+        "__int__",
+        "__invert__",
+        "__ior__",
+        "__ipow__",
+        "__irshift__",
+        "__isub__",
+        "__iter__",
+        "__iter__",
+        "__iter__",
+        "__itruediv__",
+        "__ixor__",
+        "__le__",
+        "__len__",
+        "__length_hint__",
+        "__lshift__",
+        "__lt__",
+        "__matmul__",
+        "__missing__",
+        "__mod__",
+        "__mul__",
+        "__name__",
+        "__ne__",
+        "__neg__",
+        "__next__",
+        "__or__",
+        "__pos__",
+        "__pow__",
+        "__qualname__",
+        "__radd__",
+        "__rand__",
+        "__rdivmod__",
+        "__reduce__",
+        "__reduce_ex__",
+        "__repr__",
+        "__reversed__",
+        "__rfloordiv__",
+        "__rlshift__",
+        "__rmatmul__",
+        "__rmod__",
+        "__rmul__",
+        "__ror__",
+        "__round__",
+        "__rpow__",
+        "__rrshift__",
+        "__rshift__",
+        "__rsub__",
+        "__rtruediv__",
+        "__rxor__",
+        "__set__",
+        "__set_name__",
+        "__setformat__",
+        "__setitem__",
+        "__setstate__",
+        "__sizeof__",
+        "__str__",
+        "__sub__",
+        "__subclasscheck__",
+        "__truediv__",
+        "__trunc__",
+        "__xor__",
+        # TODO
+        # "__await__",
+        # "__aiter__",
+        # "__anext__",
+        # "__aenter__",
+        # "__aexit__",
```

Comment:
I suppose this is the other PRs for async?

Reply:
Yes, next PRs are dealing with that.